addons/l10n_in_edi/models/account_edi_format.py

```diff
@@ -346,6 +361,56 @@ def _get_l10n_in_edi_line_details(self, index, line, line_tax_details, sign):
         }
 
     def _l10n_in_edi_generate_invoice_json(self, invoice):
+        def discount_group_key(line_vals):
+            return "%s-%s"%(line_vals['HsnCd'], line_vals['GstRt'])
+
+        def put_discount_on(discount_line_vals, other_line_vals):
+            discount = discount_line_vals['AssAmt'] * -1
+            discount_to_allow = other_line_vals['AssAmt']
+            if float_compare(discount_to_allow, discount, precision_digits=2) == -1:
```

Comment:
```suggestion
            if float_compare(discount_to_allow, discount, precision_digits=2) < 0:
```
might be clearer to what it does

Reply:
done